sb-common/src/main/kotlin/ru/driics/sablebot/common/model/discord/TextChannelDTO.kt

```
@@ -0,0 +1,3 @@
+package ru.driics.sablebot.common.model.discord
+
+data class TextChannelDTO(): ChannelDTO
```

detekt (1.23.8)

[warning] An empty default constructor can be removed.

(detekt.empty-blocks.EmptyDefaultConstructor)